app/views/hyrax/base/_search_results.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2 class="sr-only top-content-title"><%= t('blacklight.search.search_results_header') %></h2>
+<div class="h2 sr-only top-content-title"><%= t('blacklight.search.search_results_header') %></div>
```

Comment:
I don't think either of these lines are necessary. We don't use search constraints here.